.github/workflows/main.yml

```diff
@@ -9,6 +9,13 @@ jobs:
     
     steps:
     - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
+    - name: Run HTML-Minifier
+      run: |
+        npm i html-minifier -g
+        cp RfidShelf/index_template.html RfidShelf/index_template.full.html
+        html-minifier RfidShelf/index_template.full.html --collapse-whitespace --remove-comments --remove-optional-tags --remove-redundant-attributes --remove-script-type-attributes --remove-tag-whitespace --use-short-doctype --minify-css true --minify-js true > RfidShelf/index_template.html
```

Comment:
There is actually a difference in FileSize 5 KB vs 4.3 KB and there are few options for further optimisation.

The ESP8266 is not the fastest chip and the smaller the file size the faster is its transmission.
The second aspect: the current JavaScript implementation has very short method names and variables to save space. With the minifier it would be possible to use self-explaining long names for JS and CSS.

It is also possible to generate exactly the same file locally, but it would need to have node.js installed. If this is ok, I could add this part to extra_script of platformio.ini

Reply:
Well since the HTML is served gzipped it's only about 4.3KB send to the client anyway, which results in 80-150ms loading time for me, which seems fine to me. While this might look different from a Web Devs perspective, I have a strong opinion about having to install node.js to build something.

If performance optimization should be necessary, I would be very easy to add caching for the HTML (since it's 100% static) or to add a minifier to the gzip prozess (I just couldn't yet figure out how to do this without having to install the required python modules manually beforehand)

As for the Method/Class Names: I think it would be ok to rename them (especially s() and h() always get me confused) with the current solution in place